clients/admin-ui/src/features/consent-settings/GppConfiguration.tsx

```diff
@@ -0,0 +1,113 @@
+    <Text fontSize="sm" fontWeight="bold" lineHeight={5} color="gray.700">
+      {title}
+    </Text>
+    {children}
+  </Stack>
+);
+
+const GppConfiguration = () => {
+  const { tcf: isTcfEnabled } = useFeatures();
+  const gppSettings = useAppSelector(selectGppSettings);
+  const isEnabled = !!gppSettings.enabled;
+  const { values } = useFormikContext<{ gpp: GPPSettings }>();
+  const showMspa = !!values.gpp.us_approach;
+
+  return (
+    <SettingsBox title="Global Privacy Platform">
+      <Stack spacing={6}>
+        <FrameworkStatus name="GPP" enabled={isEnabled} />
+        {isEnabled ? (
+          <>
+            <Section title="GPP U.S.">
+              <CustomRadioGroup
+                name="gpp.us_approach"
+                variant="stacked"
+                defaultFirstSelected={false}
+                options={[
+                  {
+                    label: "Enable U.S. National",
+                    value: GPPUSApproach.NATIONAL,
+                    tooltip:
+                      "When US National is selected, Fides will present the same privacy notices to all consumers located anywhere in the United States.",
+                  },
+                  {
+                    label: "Enable U.S. State-by-State",
+                    value: GPPUSApproach.STATE,
+                    tooltip:
+                      "When state-by-state is selected, Fides will only present consent to consumers and save their preferences if they are located in a state that is supported by the GPP. The consent options presented to consumers will vary depending on the regulations in each state.",
+                  },
+                ]}
+              />
+            </Section>
+            {showMspa ? (
+              <Section title="MSPA">
+                <CustomCheckbox
+                  name="gpp.mspa_covered_transactions"
+                  label="All transactions covered by MSPA"
+                  tooltip="When selected, the Fides CMP will communicate to downstream vendors that all preferences are covered under the MSPA."
+                />
+                <CustomSwitch
+                  label="Enable MSPA service provider mode"
+                  name="gpp.mspa_service_provider_mode"
+                  variant="switchFirst"
+                  tooltip="Enable opt-out option mode if you engage or may engage in the sales or sharing of personal information, or process any information for the purpose of targeted advertising."
+                  isDisabled={values.gpp.mspa_opt_out_option_mode}
+                />
+                <CustomSwitch
+                  label="Enable MSPA opt-out option mode"
+                  name="gpp.mspa_opt_out_option_mode"
+                  variant="switchFirst"
+                  tooltip="Enable service provider mode if you do not engage in any sales or sharing of personal information."
+                  isDisabled={values.gpp.mspa_service_provider_mode}
+                />
```

Comment:
@allisonking came looking here for a reference on the descriptions i can put on the BE for these config properties, and noticed that it looks like the `tooltip` values are switched for the two options here?

Reply:
🤦‍♀️ thank you for catching that!!!

Reply:
fixed!